dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeClient.java

```diff
@@ -920,7 +920,6 @@ private Challenge prepareHttpChallenge(Authorization auth) throws AcmeCaExceptio
 				+ challenge.getAuthorization() + "'.");
 		return challenge;
 	}
-
 	/**
 	 * Revoke a certificate using an existing account on the ACME server. If the
 	 * account key pair cannot be found, we will fail.
```

Comment:
Probably should update the comment to say this uses the configured directory URI.